test/integration/src/test/java/io/pravega/test/integration/WatermarkingTest.java

```diff
@@ -514,6 +482,190 @@ public void progressingWatermarkWithWriterTimeouts() throws Exception {
         assertNull(nullMark);
     }
 
+    @Test
+    public void test() {
+        String scopeName = "testScope";
+        String streamName = "testStreamstream";
+        String readerGroupName = "testReaderGroup";
+        System.out.println(String.join("", "prn::/scope:", scopeName, "/reader-group:", readerGroupName, ",", "READ"));
+    }
+
+    @SneakyThrows
+    @Test
+    public void watermarkingWithAuth() {
+        String writerUserName = "writer";
+        String readerUserName = "reader";
+        String userPassword = "test-password";
+
+        String scopeName = "testScope";
+        String streamName = "testStreamstream";
+        String readerGroupName = "testReaderGroup";
+        String readerName = "testReader";
+
+        final Map<String, String> passwordInputFileEntries = new HashMap<>();
+        passwordInputFileEntries.put(writerUserName, String.join(";",
+                "prn::/,READ_UPDATE",
+                String.join("", "prn::/scope:", scopeName, ",", "READ_UPDATE"),
+                String.join("", "prn::/scope:", scopeName, "/stream:", streamName, ",", "READ_UPDATE")
+        ));
+
+        passwordInputFileEntries.put(readerUserName, String.join(";",
+                "prn::/,READ",
+                String.join("", "prn::/scope:", scopeName, ",", "READ"),
+                String.join("", "prn::/scope:", scopeName, "/stream:", streamName, ",", "READ"),
+                String.join("", "prn::/scope:", scopeName, "/reader-group:", readerGroupName, ",", "READ")
+        ));
+
+        @Cleanup
+        final ClusterWrapper cluster = ClusterWrapper.builder()
+                .authEnabled(true)
+                .tokenSigningKeyBasis("secret").tokenTtlInSeconds(600)
+                .rgWritesWithReadPermEnabled(true)
```

Comment:
I believe the rgWritesWithReadPermEnabled could have been set to false with the same results in the test, is that true?  Just curious.

Reply:
Yes, this flag'd be irrelevant to the issue at hand. Just that not enabling it'd require additional permissions to be assigned to the `reader`, which'd have made it slightly harder to understand this test. 

This test has now relocated to `watermarkingAuthorizationTest()` method now, FYI.